src/components/atoms/SfSearchBar/SfSearchBar.html

```diff
@@ -0,0 +1,23 @@
+<div class="search-bar">
```

Comment:
I think `sf-` prefix should be used here, we're just not using prefix for SCSS vars, but class names stills with the prefix (which is a good practice for compatibility).